signing_service/signing_service/tests/tests.py

```diff
@@ -174,13 +188,50 @@ def test_that_argument_parser_should_parse_correct_input_and_use_default_values(
     def test_that_argument_parser_should_fail_if_port_cannot_be_casted_to_int(self):
         sys.argv += ['127.0.0.1', self.concent_public_key_encoded, '--initial_reconnect_delay', '1', '--concent-cluster-port', 'abc']
 
-        with self.assertRaises(SystemExit):
-            _parse_arguments()
+        with mock.patch.dict(os.environ, {
+            'SENTRY_DSN': self.sentry_dsn,
+            'ETHEREUM_PRIVATE_KEY': b64encode(self.ethereum_private_key).decode('ascii'),
+        }):
+            with self.assertRaises(SystemExit):
+                _parse_arguments()
 
     def test_that_argument_parser_should_fail_if_parameters_are_missing(self):
-        with self.assertRaises(SystemExit):
+
+        with mock.patch.dict(os.environ, {
+            'SENTRY_DSN': self.sentry_dsn,
+            'ETHEREUM_PRIVATE_KEY': b64encode(self.ethereum_private_key).decode('ascii'),
+        }):
+            with self.assertRaises(SystemExit):
+                _parse_arguments()
+
+    def test_that_argument_parser_should_parse_correct_secrets_from_command_line(self):
+        sys.argv += ['127.0.0.1', self.concent_public_key_encoded, '--sentry-dsn', self.sentry_dsn, '--ethereum-private-key', b64encode(self.ethereum_private_key).decode('ascii')]
+
+        args = _parse_arguments()
+
+        self.assertEqual(args.sentry_dsn, self.sentry_dsn)
+        self.assertEqual(args.ethereum_private_key, self.ethereum_private_key)
+
+    def test_that_argument_parses_should_fail_if_file_with_secrets_is_missing(self):
+        sys.argv += ['127.0.0.1', self.concent_public_key_encoded, '--sentry-dsn-path', '/not_existing_path/file.txt']
+        with self.assertRaises(FileNotFoundError):
             _parse_arguments()
 
+    def test_that_argument_parser_should_parse_correct_sentry_dsn_and_ethereum_private_key_if_files_exist(self):
```

Comment:
files are not deleted after test. Please fix it

Reply:
Oh, I completely forgot about it. Done